app/code/Magento/Customer/Test/Mftf/ActionGroup/StorefrontClickShowPasswordActionGroup.xml

```diff
@@ -0,0 +1,21 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+ /**
+  * Copyright © Magento, Inc. All rights reserved.
+  * See COPYING.txt for license details.
+  */
+-->
+
+<actionGroups xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+              xsi:noNamespaceSchemaLocation="urn:magento:mftf:Test/etc/actionGroupSchema.xsd">
+    <actionGroup name="StorefrontClickShowPasswordActionGroup">
```

Comment:
This action group is basically a proxy to `click` MFTF operator. I'd avoid using selector as an argument to make the action group safer to extend and easier to maintain (but yes, there might be a need to introduce several similar action groups).

This should be applied to other action groups as well

Reply:
@sivaschenko Instead of passing selector as an argument to action groups. Shall I directly reference the selectors in the action group itself?  Is that fine?

Also what I understand from the comment is, It's not the best practice to pass selector as an argument to action groups. Right? Please correct me If I misunderstood anything. 

Thanks.

Reply:
@ajithkumar-maragathavel exactly, it's better to reference the element directly from the action group